app/presenters/tree_builder.rb

```diff
@@ -257,9 +257,15 @@ def x_build_node(object, pid, options)
     node = x_build_single_node(object, pid, options)
 
     # Process the node's children
+    load_children = if object.kind_of?(Struct)
```

Comment:
Withdrawing my objection here, I looked into it, it seems like that `object[:load_children]` that is already there should have had a `object === Hash` in front of it, since `object` can be any kind of AREL model.

So adding another branch for structs ..sounds about right :).